iis-wf/iis-wf-referenceextraction/src/main/resources/eu/dnetlib/iis/wf/referenceextraction/project/tara/oozie_app/workflow.xml

```diff
@@ -103,6 +103,7 @@
                 --conf spark.sql.queryExecutionListeners=${spark2SqlQueryExecutionListeners}
                 --conf spark.yarn.historyServer.address=${spark2YarnHistoryServerAddress}
                 --conf spark.eventLog.dir=${nameNode}${spark2EventLogDir}
+                --conf spark.sql.shuffle.partitions=2880
```

Comment:
Let's define `2880` as an explicit workflow parameter. Probably with some generic name (`sparkSqlShufflePartitions`?) which could be potentially used also in other workflows whenever needed. Similarly to the set of other spark related params such as: `sparkExecutorMemory`, `sparkExecutorCores` etc.

Since this value is bound to the cluster setup then I would prefer to define the value in `default-config.xml` file already sharing cluster-wide configuration so it could be used in all the workflows accepting this parameter.

Reply:
Ok, I've used `sparkSqlShufflePartitions`.